core/templates/pages/exploration-player-page/learner-experience/tutor-card.directive.ts

```diff
@@ -205,6 +206,15 @@ angular.module('oppia').directive('tutorCard', [
             $scope.waitingForOppiaFeedback = false;
             $scope.windowDimensionsService = WindowDimensionsService;
             $scope.isIframed = UrlService.isIframed();
+            $scope.audioBarIsExpandedAndOnMobileDevice = false;
+            $scope.$watch(function() {
+              return AudioBarStatusService.isAudioBarExpanded();
```

Comment:
instead of `audioBarIsExpandedAndOnMobileDevice` can we create a function `isAudioBarExpandedInMobileDevice` inside `AudioBarStatusService` and link that function in the scope here? (This is just to avoid dedicated watcher.)

Reply:
DOne